pages/portfolio.html

```diff
@@ -5,8 +5,63 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="/styles/style.css" rel="stylesheet" />
 </head>
 <body>
-  
+  <header>
+    <h1 class="header">PORTFOLIO</h1>
+    <nav>
+      <ul>
+        <li><a href="index.html">HOME</a></li>
+        <li><a href="about.html">ABOUT</a></li>
+        <li><a href="art.html">ART</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main class="main">
+    <section>
+      <h2>Past Projects</h2>
+    </section>
+    <section>
+      <br>
+      <br>
```

Comment:
Style: You should avoid the `br` tag for vertical spacing.  Instead you should use `margin` or `padding` in CSS.